src/utils/auth-handlers/example-cognito-config.js

```diff
@@ -22,8 +22,8 @@ export default {
   },
   Storage: {
     AWSS3: {
-      bucket: "invalid", //Your bucket name;
-      region: "invalid", //Specify the region your bucket was created in;
+      bucket: "annotation-tool-storage", //Your bucket name;
+      region: "ca-central-1", //Specify the region your bucket was created in;
```

Comment:
I think these were committed accidentally

Reply:
Again i should put it in git ignore